install/migrations/update_9.5.x_to_10.0.0/databases.php

```diff
@@ -0,0 +1,164 @@
+ *
+ * GLPI is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with GLPI. If not, see <http://www.gnu.org/licenses/>.
+ * ---------------------------------------------------------------------
+ */
+/**
+ * @var DB $DB
+ * @var Migration $migration
+ * @var array $ADDTODISPLAYPREF
+ */
+
+$default_charset = DBConnection::getDefaultCharset();
+$default_collation = DBConnection::getDefaultCollation();
+
+if (!$DB->tableExists('glpi_databasetypes')) {
+   $query = "CREATE TABLE `glpi_databasetypes` (
+         `id` int NOT NULL AUTO_INCREMENT,
+         `name` varchar(255) DEFAULT NULL,
+         `comment` text,
+         `date_mod` timestamp NULL DEFAULT NULL,
+         `date_creation` timestamp NULL DEFAULT NULL,
+         PRIMARY KEY (`id`),
+         KEY `name` (`name`),
+         KEY `date_mod` (`date_mod`),
+         KEY `date_creation` (`date_creation`)
+      ) ENGINE = InnoDB ROW_FORMAT = DYNAMIC DEFAULT CHARSET = {$default_charset} COLLATE = {$default_collation};";
+   $DB->queryOrDie($query, "10.0 add table glpi_databasetypes");
+}
+
+if (!$DB->tableExists('glpi_databasecategories')) {
+   $query = "CREATE TABLE `glpi_databasecategories` (
+         `id` int NOT NULL AUTO_INCREMENT,
+         `name` varchar(255) DEFAULT NULL,
+         `comment` text,
+         `date_mod` timestamp NULL DEFAULT NULL,
+         `date_creation` timestamp NULL DEFAULT NULL,
+         PRIMARY KEY (`id`),
+         KEY `name` (`name`),
+         KEY `date_mod` (`date_mod`),
+         KEY `date_creation` (`date_creation`)
+      ) ENGINE = InnoDB ROW_FORMAT = DYNAMIC DEFAULT CHARSET = {$default_charset} COLLATE = {$default_collation};";
+   $DB->queryOrDie($query, "10.0 add table glpi_databasecategoriess");
+}
+
+if (!$DB->tableExists('glpi_databases')) {
+   $query = "CREATE TABLE `glpi_databases` (
+         `id` int NOT NULL AUTO_INCREMENT,
+         `entities_id` int NOT NULL DEFAULT '0',
+         `is_recursive` tinyint NOT NULL DEFAULT '0',
+         `name` varchar(255) NOT NULL DEFAULT '',
+         `version` varchar(255) NOT NULL DEFAULT '',
+         `is_active` tinyint NOT NULL DEFAULT '0',
+         `is_deleted` tinyint NOT NULL DEFAULT '0',
+         `databasetypes_id` int NOT NULL DEFAULT '0',
+         `databasecategories_id` int NOT NULL DEFAULT '0',
+         `locations_id` int NOT NULL DEFAULT '0',
+         `manufacturers_id` int NOT NULL DEFAULT '0',
+         `suppliers_id` int NOT NULL DEFAULT '0',
+         `users_id_tech` int NOT NULL DEFAULT '0',
+         `groups_id_tech` int NOT NULL DEFAULT '0',
+         `date_mod` timestamp NULL DEFAULT NULL,
+         `date_creation` timestamp NULL DEFAULT NULL,
+         `date_lastboot` timestamp NULL DEFAULT NULL,
```

Comment:
I guess a database has always an instance. Is this `date_lastboot` is relevant if we already have one for each instances ?

Reply:
In original specifications; last boot date is set only on databases; and not at all on instances; but maybe should it be set only on instances indeed. Any thoughts @orthagh @stonebuzz?

Reply:
I've dropped that lastboot field